### ISSUE TITLE
[Bug] Timewindow on chart widgets does not recover default range when changing datasource after making zoom

### ISSUE THREAD

**Issue by amarco2** [thingsboard/thingsboard#8971]
**Describe the bug**
When displaying telemetry data from an entity alias that may change (for example having a list of entities, and displaying the temperature of the current selected entity on the list, by setting an entity state), if we make a zoom on the chart on the data, a double click on the chart returns to the original range. But if we change the source alias (selecting another entity on the list) after making the zoom, the timewindow does not return to the original range on a double click.

I guess this is because after changing the datasource, the query takes the current values of the time window instead of the default values savec for the chart widget, thus the double click always return to show not the default timewindow but all the data available, what does not match each other if we make a zoom before retrieving the data for the new datasource.

**Your Server Environment**

* demo.thingsboard.io
* own setup
  * docker deployment
  * ThingsBoard CE 3.4.4 (suppose the same on any version)
  * Debian

**Your Client Environment**

**Desktop (please complete the following information):**

* OS: Windows 10
* Browser Edge
* Version 114.0.1823.82 

**To Reproduce**
1. Configure a dashboard with an entity list from several entities with some telemetry data.
2. Add an action on the list for setting the selected entity as entity state "selected_entity"
3. Add a chart widget to show telemetry from state entity "selected_entity" and define the window for the last xxx days or whatever range you have data
4. Open the dashboard and select an entity from the list
5. Make a zoom on the chart or change the time window
6. Select another entity on the list
7. Double click on the chart to restore the original range saved on the widget (last xxx days)
8. See the error

**Expected behavior**
Restore the original time window range (last xxx days) after step 7

**Comment by mykhailokornieiev** [comment]
Hi. Chart widgets of 3.4.4 versions are currently deprecated, so they won't be changed/fixed. Please check this behavior on the latest version. If it still occurs, please create a new ticket with a full description of your request, with full detailed steps to reproduce.
